modules/0_git/test/main.cpp

```diff
@@ -7,7 +7,7 @@
 TEST(git, say_hello) {
     myspace::A a;
     EXPECT_EQ(myspace::func(a), "Hello, Nizhny!");
-    EXPECT_EQ(func(a), "Hello, World!");
+    EXPECT_EQ(func(a), "Hello, Nizhny!");
```

Comment:
please start from a master branch

Reply:
i swear i did

Reply:
oh, i see, it has only been restored remotely, and the local file has remained """corrupted""" so-to-say, and got pushed as an ancestor from local master. Modified the file locally, i hope this will not be an issue anymore

Reply:
please do the same for coins task

Reply:
did i not already?

Reply:
https://github.com/dkurt/openvino_practice/pull/162/files

Reply:
oh, that's what you mean
will do